packages/nextra-theme-docs/src/utils/normalize-pages.ts

```diff
@@ -14,15 +15,15 @@ function extendMeta(
   return Object.assign({}, fallback, meta, { theme })
 }
 
-export interface Item extends Omit<PageMapItem, 'children'> {
+export interface Item extends MdxFile {
```

Comment:
which type `Item/PageItem/MenuItem/DocsItem` should extend now? maybe `Folder & MdxFile`? Since `MdxFile` doesn't have `children` prop